### PR TITLE
RAW content viewing

### PR DESCRIPTION
**Pull request by ghost**
**What does this PR?**

Show content of post and comments in raw format when user enter ?raw in post link.

**Steps to reproduce**

1. Open any post and append ?raw at the end of the link of that post.

**Issue number** 

fixes #[923](https://github.com/ecency/ecency-vision/issues/923)

**Screenshot / Video**

https://user-images.githubusercontent.com/106739598/207622181-b7d84635-72b9-42a6-b1c4-e038d5bcd951.mp4

### CODE REVIEW

**Comment by feruzm** [comment]
why this is closed, by accident perhaps @MTSaeed ?

File: src/common/components/discussion/index.tsx [hunk 1]
```diff
@@ -66,23 +66,38 @@ import { ProfilePopover } from "../profile-popover";
...
   }
 
+  componentDidUpdate(
+    prevProps: Readonly<ItemBodyProps>,
+    prevState: Readonly<{}>,
+    snapshot?: any
+  ): void {
+    // console.log("Componentdid update");
+    // console.log(this.props);
+  }
```
Comment (dkildar) [inline src/common/components/discussion/index.tsx:84]: Why it needs?

File: src/common/components/discussion/index.tsx [hunk 2]
```diff
@@ -66,23 +66,38 @@ import { ProfilePopover } from "../profile-popover";
...
 
     const renderedBody = { __html: renderPostBody(entry.body, false, global.canUseWebp) };
-
     return (
-      <div
-        className="item-body markdown-view mini-markdown"
-        dangerouslySetInnerHTML={renderedBody}
-      />
+      <>
+        {!this.props.isRawContent ? (
```
Comment (dkildar) [inline src/common/components/discussion/index.tsx:92]: Please, always check positive condition because negative one confusing

Comment (MTSaeed) [reply src/common/components/discussion/index.tsx:92]: > why this is closed, by accident perhaps @MTSaeed ?
 @feruzm I noticed that this PR had faulty commits which were done using force push so I closed this PR and created a new branch to remain out of trouble. I will create a new PR for this task and ask other developers for suggestions about the issue that we discussed in our last call.

File: src/common/components/discussion/index.tsx (hunk 1 above)
Comment (MTSaeed) [reply src/common/components/discussion/index.tsx:84]: @dkildar  As discussed with Feruz this issue is on hold and we are still developing. I have assigned it back to me and will assign the PR to you and Feruz when its ready.

**Comment by MTSaeed** [comment]
I created a new branch and assigned PR for review to @feruzm . I would suggest you to close this PR and review the new ones.
Thanks

**Comment by feruzm** [comment]
closing in favor of #1175

**Pull request closed without merging**